src/Components/SingleReview.jsx

```diff
@@ -14,8 +15,9 @@ export default function SingleReview({ isLoading, setIsLoading }) {
     });
   }, []);
 
+
   const displaySingleReview = (review) => {
-    const postedAt = review.created_at.toString()
+    const postedAt = review.created_at
```

Comment:
We never use this variable - I would either delete it or use it. The same goes for any states we don't use